package.json

```diff
@@ -83,6 +99,22 @@
 				{
 					"command": "metaformTreeDataProvider.updatePdfExportTheme",
 					"when": "view == metaformTreeDataProvider && viewItem == pdfexporttheme"
+				},
+				{
+					"command": "metaformTreeDataProvider.createEmailNotification",
+					"when": "view == metaformTreeDataProvider && viewItem == emailnotifications"
```

Comment:
Should this be:
```suggestion
					"when": "view == metaformTreeDataProvider && viewItem == emailnotification"
```

Reply:
No, it would move the creation of email notifications to single email notificatino menu item and thus prevent creation of the first notification